packages/service/models/request.ts

```diff
@@ -0,0 +1,59 @@
+import { z } from 'zod';
+import { RequestId, UserId, CourseId, FileReference, type RequestType } from './util.js';
+
+export const RequestDetails = z.object({
+    reason: z.string(),
+    proof: z.array(FileReference),
```

Comment:
```suggestion
    proof: z.array(z.file()),
```

还可以设置MIME

<img width="705" height="401" alt="Image" src="https://github.com/user-attachments/assets/ac11c92b-695f-4531-8f7d-69917b9aadb7" />